llzk-sys/build-support/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "llzk-sys-build-support"
+version = "0.1.0"
+edition = "2024"
+keywords = ["llzk", "mlir"]
+categories = ["development-tools::build-utils"]
+description = "Helper library for building the LLZK Rust bindings."
+repository = "https://github.com/Veridise/llzk-rs"
+readme = "README.md"
```

Comment:
This readme doesn't exist (I'm a little surprised it doesn't whine at you for that, or maybe it does). It would be nice if there was a readme for this, actually.

Reply:
still pending?

Reply:
yeah since moving the build support module to a separate crate happens in this PR